### DISCUSSION TITLE
Eufy streaming service down?

### DISCUSSION THREAD

**Discussion by PeterAlexander75** [samemory/homebridge-eufy-security-control#211] [General]
Is anyone else finding that https://mysecurity.eufylife.com/#/camera is presently not working reliably at all, thus impacting streaming?

**Comment by lenoxys** [comment]
Hey,

I'm logged into the website but not able to start a stream but using App on my Cell Networks (outside home) streaming work fine.

**Reply by PeterAlexander75** [reply]
The app seems to work fine. The website stream does not for me, two different “guest” accounts and master account tested. No joy.

**Reply by lenoxys** [reply]
It looks like Eufy plugin is not impacted but since we don't have a stable stream at all I can't say.

**Comment by PeterAlexander75** [comment]
Thanks. Just wanted to check it wasn’t my account that was the issue. Guess we’ll wait and see if eufy fix the issue.

**Comment by PeterAlexander75** [comment]
Still down this morning. Frustrating.
